hazelcast-jet-all/src/main/java/com/hazelcast/jet/server/JetCommandLine.java

```diff
@@ -712,4 +814,373 @@ static Throwable peel(Throwable e) {
         }
     }
 
+    /**
+     * A parser for SQL-like inputs. Commands are terminated with a semicolon.
+     * It is mainly taken from
+     *
+     * @see <a href="https://github.com/julianhyde/sqlline/blob/master/src/main/java/sqlline/SqlLineParser.java">
+     * SqlLineParser</a>
+     * which is licensed under the BSD-3-Clause License
+     */
+    private static final class MultilineParser extends DefaultParser {
+
+        private MultilineParser() {
+        }
+
+        @Override
+        public ParsedLine parse(String line, int cursor, Parser.ParseContext context) throws SyntaxError {
+            super.setQuoteChars(new char[]{'\'', '"'});
+            super.setEofOnUnclosedQuote(true);
+            stateCheck(line, cursor);
+            return new ArgumentList(line, Collections.emptyList(), -1, -1,
+                    cursor, "'", -1, -1);
+        }
+
+        private void stateCheck(String line, int cursor) {
```

Comment:
When you submit this command, it executes it, even though it doesn't end with `;`:
```
/**/
```
Seems that after a multi-line comment it's treated as an end of command.

Also this:
```
--;
```

Reply:
I solved these issues.